Library/Homebrew/cask/lib/hbc/installer.rb

```diff
@@ -87,7 +87,7 @@ def summary
     s = if MacOS.version >= :lion && !ENV["HOMEBREW_NO_EMOJI"]
           (ENV["HOMEBREW_INSTALL_BADGE"] || "\xf0\x9f\x8d\xba") + "  "
         else
-          "#{Hbc::Utils::Tty.blue.bold}==>#{Hbc::Utils::Tty.reset.bold} Success!#{Hbc::Utils::Tty.reset} "
+          "#{Tty.blue}==>#{Tty.reset} #{Tty.white}Success! #{Tty.reset}"
```

Comment:
Very minor nit, but I think the trailing space should remain after the final `#{Tty.reset}`, not moved before it.